File: baseui/src/main/java/com/example/baseui/MainActivity.java
package com.example.baseui;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    private CircleProgressView mCircleProgressView;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_main);

        mCircleProgressView = findViewById(R.id.progress_view);
        mCircleProgressView.setValue("80",100);
    }
}

File: app/src/main/java/com/wtt/composetest/MainActivity.kt
package com.wtt.composetest

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.wtt.composetest.databinding.MainBinding
import com.wtt.composetest.repository.hilt.SimpleInjectActivity
import com.wtt.composetest.socket.SocketActivity

class MainActivity : AppCompatActivity() {
    private lateinit var binding: MainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = MainBinding.inflate(layoutInflater)
        setContentView(binding.root)


        binding.tvJumpSocket.setOnClickListener {
            startActivity(
                Intent(this, SocketActivity::class.java)
            )
        }
        binding.tvJumpHilt.setOnClickListener {
            startActivity(
                Intent(this, SimpleInjectActivity::class.java)
            )
        }

    }
}
